src/Algebra/Module/Morphism/Linear/Properties.agda

```diff
@@ -0,0 +1,206 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of linear maps.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra                   using (CommutativeRing)
+open import Algebra.Module            using (Module)
+open import Level                     using (Level; _⊔_)
+
+module Algebra.Module.Morphism.Linear.Properties
+  {r ℓr m ℓm : Level}
+  {ring      : CommutativeRing r ℓr}
+  (modA modB  : Module ring m ℓm)
+  where
+
+import Algebra.Module.Properties          as Properties
+import Function.Definitions
+import Relation.Binary.Reasoning.Setoid   as Reasoning
+import Algebra.Module.Morphism.Structures as MorphismStructures
+
+open import Agda.Builtin.Sigma
+open import Axiom.DoubleNegationElimination
+open import Data.List
+open import Data.Product
+  using (Σ; _,_; ∃; Σ-syntax; ∃-syntax; _×_)
+open import Relation.Nullary          using (¬_)
+open import Relation.Nullary.Negation using (contraposition)
+
+open MorphismStructures.ModuleMorphisms modA modB
+open Module modA
```

Comment:
Have you considered doing
```agda
module A = Module modA
module B = Module modB
```
instead, and then using qualified names? agda-categories uses this later style, and it largely quite readable.

Note that I'm not saying that you *have* to do this, merely recommending that you give it an honest try, and see if you like it or not.

Reply:
Doesn't that collide with my use of "A" and "B" for the carrier types of those modules?

Reply:
Turns out that no, it doesn't: different name spaces for symbols and modules. So you can name them both A and B without problem.

Reply:
Okay, I switched to your suggested notation.
I think, visually, I prefer the operators w/ superscript "M" replaced by "A"/"B", but I _really_ like getting rid of that long list of renames!

**These changes included in latest push and ready for review.**